src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_PRICE = 40;
+
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = DAY_PRICE * days;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
```

Comment:
we can simplify the conditions
```suggestion

  if (days >= LONG_TERM) {
    return basePrice - LONG_TERM_DISCOUNT;
  }

  if (days >= SHORT_TERM ) {
    return basePrice - SHORT_TERM_DISCOUNT;
  }

```